lib/rex/proto/http/response.rb

```diff
@@ -108,7 +108,7 @@ def get_xml_document
   #
   # @return [Hash]
   def get_json_document
-    json = []
+    json = {}
```

Comment:
It's new :-) Added about a month ago:
https://github.com/rapid7/metasploit-framework/pull/6118